content/lua/concepts/mathematical-library/terms/asin/asin.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Lua .asin() Method
+---
+
+## Syntax
+
+````lua
+math.asin(x)
+
+### Description
```

Comment:
```suggestion
```

Reply:
the first content in the body should be the text of the description, we do not use a header at the top